### DISCUSSION TITLE
Unable to update GPG key

### DISCUSSION THREAD

**Discussion by Jakuje** [community/community#21851] [New to GitHub]
<p>I tried to add GPG key to my account, but since yesterday I am getting the error</p>
<p>&gt; We got an error doing that.</p>
<p>Is it just me or there is something broken on GItHub side?</p>
<p>I recently allowed 2 factor authentication, but for this action I am asked only for my password.</p>

**Comment by Jakuje** [comment, marked as answer]
<p>Now it works as expected … at least after I entered the “sudo” mode by adding SSH key.</p>

**Comment by dighbijoy** [comment]
<p>I am facing the same problem. But the ‘Sudo’ mode which just asks you to confirm your password is not working for me. Will you be able to help out in any way?</p>

**Reply by HorizonChaser** [reply]
I've just encountered the same problem, which was caused by **a subkey with secp256k1 algorithm**. After removing that subkey, the exported public key can be successfully imported😀 

But according to the [doc](https://docs.github.com/en/authentication/managing-commit-signature-verification/checking-for-existing-gpg-keys), GitHub actually supports ECDSA algorithm... That's a little weird :(
